.vscode/launch.json

```diff
@@ -5,15 +5,16 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": "(Windows) Launch",
-            "type": "cppvsdbg",
+            "name": "Debug",
```

Comment:
Probably not cleaned up yet. We should probably copy this instead of changing the existing one. I think these are probably changes to get debugging working on linux

Reply:
yeah will write one to be cross platform. LLDB might not need much changes on linux too, have to check first though

Reply:
This article seems to indicate that cppvsdbg works better on windows but that clearly wont work on other platforms https://www.forrestthewoods.com/blog/how-to-debug-rust-with-visual-studio-code/

Reply:
@Kethku yeah we just need something that could give you step through code, and variable inspection. LLDB is decent, works in vscode and is cross platform too

Reply:
No reason not to have both :) Vscode supports multiple configurations in this file

Reply:
yeah sure why not. i initially didn't mean to remove your Windows config anyway idk why the hell i did that